_filters/contract-footer.lua

```diff
@@ -26,14 +26,22 @@ function Div(el)
       local next_url = el.attributes["next-url"]
       local next_contract = el.attributes["next-contract"] or "Next contract"
       local next_text = "Next"
+
+      local version = el.attributes["version"] or "latest"
```

Comment:
I think it would be better not to show the version if it's missing from the markdown, "latest" can be misleading.

Reply:
... or it could be "unknown"